onepassword/data_source_onepassword_item.go

```diff
@@ -177,6 +211,22 @@ func dataSourceOnepasswordItemRead(ctx context.Context, data *schema.ResourceDat
 		}
 	}
 
+	dataFiles := []interface{}{}
```

Comment:
Could you please create struct instead of interface and describe all the properties there? Might be something like `FileSchema`

Reply:
I did that, but it seems to be a bit convoluted since we need to have the properties with terraform naming in the ResourceData, but the struct needs to be public for the data to come across (at least that is what I saw when fixing...)

also, there is prior art wrt `[]interface{}{}` in the datasource.

But less dynamism is probably a good thing after all.